lib/webpack/util.js

```diff
@@ -0,0 +1,27 @@
+const { readFileSync } = require('fs')
+const { resolve, dirname } = require('path')
+
+const mdExplodeIncludes = exports.mdExplodeIncludes = ({ cwd, src }) => {
+  const deps = []
+
+  return {
+    explodedSrc: src.replace(/<!-- *include +(.*?) *-->/g, (match, path) => {
+      try {
+        const absolutePath = resolve(cwd, path)
+        const content = readFileSync(absolutePath, 'utf8')
```

Comment:
Using a unified convention at the same project. either that all are `await readFile` or all are `readFileSync `

Reply:
I have made this loader asynchronous. And it took me a really really long time to complete the `asyncReplace` function. I am so glad that I did this in such an elegant way. 

Also it took me really long to debug the async loader as I forgot to call the callback when the loader hit the cache. :sob: 

It is using readFile now.